### PR TITLE
[stable/gemini] Add a few features to the chart

### PR DESCRIPTION
**Pull request by h4wkmoon**
**Why This PR?**
This PR adds a few features to the chart: tolerations, nodeSelector, pod labels & annotations.

**Changes**
Changes proposed in this pull request:

* Tolerations
* nodeSelector
* additionnalPodLabels
* additionnalPodAnnotations

**Checklist:**

* [x] I have included the name of the chart in the title of this PR in square brackets i.e. `[stable/goldilocks]`.
* [x] I have updated the chart version in `Chart.yaml` following Semantic Versioning.
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] Any new values have been added to the README for the Chart, or `helm-docs --sort-values-order=file` has been run for the charts that support it.

### CODE REVIEW

File: stable/gemini/README.md [hunk 1]
```diff
@@ -43,3 +43,8 @@ before upgrading, and add `--skip-crds` when running `helm install`.
 | rbac.serviceAccountName | string | `nil` |  |
 | verbosity | int | `5` | How verbose the controller logs should be |
 | resources | object | `{"limits":{"cpu":"200m","memory":"512Mi"},"requests":{"cpu":"25m","memory":"64Mi"}}` | The resources block for the controller pods |
+| tolerations | list | `[]` | Taint tolerations for nodes |
+| nodeSelector | object | `{}` | Select nodes to deploy which matches the following labels |
+| affinity | object | `{}` | Pod affinity and pod anti-affinity allow you to specify rules about how pods should be placed relative to other pods. |
+| additionalpodLabels | object | `{}` | Additional labels added on pod |
```
Comment (aslafy-z) [inline stable/gemini/README.md]:
```suggestion
| additionalPodLabels | object | `{}` | Additional labels added on pod |
```

File: stable/gemini/values.yaml [hunk 2]
```diff
@@ -23,3 +23,33 @@ resources:
...
+#    requiredDuringSchedulingIgnoredDuringExecution:
+#      nodeSelectorTerms:
+#      - matchExpressions:
+#        - key: disktype
+#          operator: In
+#          values:
+#          - ssd
+
+# additionalpodLabels -- Additional labels added on pod
+additionalpodLabels: {}
```
Comment (aslafy-z) [inline stable/gemini/values.yaml]:
```suggestion
additionalPodLabels: {}
```

**Review by aslafy-z — CHANGES_REQUESTED** [review]
Here's a little typo.

File: stable/gemini/values.yaml [hunk 3]
```diff
@@ -23,3 +23,33 @@ resources:
...
+#  nodeAffinity:
+#    requiredDuringSchedulingIgnoredDuringExecution:
+#      nodeSelectorTerms:
+#      - matchExpressions:
+#        - key: disktype
+#          operator: In
+#          values:
+#          - ssd
+
+# additionalpodLabels -- Additional labels added on pod
```
Comment (aslafy-z) [inline stable/gemini/values.yaml]:
```suggestion
# additionalPodLabels -- Additional labels added on pod
```

**Review by rbren — APPROVED** [review]

**Pull request merged**